src/platform/Darwin/CHIPPlatformConfig.h

```diff
@@ -66,5 +75,18 @@ extern "C" int __cxa_atexit(void (*f)(void *), void * p, void * d);
 #endif // CHIP_CONFIG_BDX_MAX_NUM_TRANSFERS
 
 #ifndef CHIP_CONFIG_KVS_PATH
+#if TARGET_OS_IPHONE
+#define CHIP_CONFIG_KVS_PATH "chip.store"
+#else
 #define CHIP_CONFIG_KVS_PATH "/tmp/chip_kvs"
+#endif // TARGET_OS_IPHONE
 #endif // CHIP_CONFIG_KVS_PATH
+
+#define CHIP_SYSTEM_CONFIG_PACKETBUFFER_POOL_SIZE 0
+
+// The session pool size limits how many subscriptions we can have live at
+// once.  Home supports up to 1000 accessories, and we subscribe to all of them,
+// so we need to make sure the pool is big enough for that.
+#define CHIP_CONFIG_SECURE_SESSION_POOL_SIZE 1000
```

Comment:
Do we even need to hard-limit the number of secure sessions in this way? `SecureSessionTable::EvictAndAllocate` does this rather complicated eviction logic... can we just remove the limit and compile out the eviction code entirely? (Probably something for a follow-up change)

Reply:
Followup at best; the secure session situation is complicated.